src/app/(tabs)/search/styles.ts

```diff
@@ -62,7 +64,7 @@ const styles = StyleSheet.create({
   clearAll: {
     color: colors.gwnOrange,
     fontSize: 12,
-    fontWeight: '400',
+    fontWeight: '600',
```

Comment:
Setting the `fontWeight` breaks on android, so you should just round the fontWeight up to 700 and use one of the global styles (Manrope-Bold has a font weight of 700). This looks like it should be `globalStyles.subHeading2Bold` (you can use `...globalStyles.subHeading2Bold` to add all of its attributes into the `clearAll` object